reference_guide/intellij_artifacts.md

```diff
@@ -0,0 +1,111 @@
+---
+title: IntelliJ Platform Artifacts Repository
+---
+
+JetBrains maintains a public repository that hosts artifacts related to the IntelliJ Platform, such as binaries and source code. This 
+repository makes artifacts more accessible for plugin authors.
+
+The IntelliJ Platform Artifacts Repository contains:
+* [Releases repository](https://www.jetbrains.com/intellij-repository/releases/) for release and EAP versions.
+* [Snapshots repository](https://www.jetbrains.com/intellij-repository/snapshots/) for nightly successful 
+  builds of each branch.
+  
+More artifacts will be hosted in the future.
+
+[Dependencies](https://jetbrains.bintray.com/intellij-third-party-dependencies) of individual modules from the IntelliJ
+Platform are hosted at the bintray repository. These artifacts should not be used directly, but a link to this 
+repository should be added to pom.xml/build.gradle files if individual modules from the IntelliJ Platform Artifacts Repository are used.    
+
+## How to Use the IntelliJ Platform Artifacts Repository
+Artifacts in the repositories are utilized by adding information to a project's build.gradle file. See
+the [gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin) for more information about
+Gradle support. 
+
+There are two types of information needed to use a repository:
+1. Specify the corresponding repository URL for the artifact.
+2. Specify the Maven coordinates for the artifact. 
+ 
+### Add the Repository URL 
+The corresponding URL for the desired artifact needs to be added to a Maven or Gradle script:
+* For release or EAP versions, use https://www.jetbrains.com/intellij-repository/releases 
+* For snapshot or EAP snapshots, use https://www.jetbrains.com/intellij-repository/snapshots
+* For dependencies of individual modules from the IntelliJ Platform, use https://jetbrains.bintray.com/intellij-third-party-dependencies 
```

Comment:
I would replace 'use' by 'also use' in the third item. As stated above, `intellij-third-party-dependencies` is an additional repository, and if you want to use individual modules you still take them from `releases` or `snapshot` repository.